core/jazz_t-vault/config/global-config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "CREATE_SAFE_REQUIRED_FIELDS": [
+    "safeName",
```

Comment:
update these as described in swagger - 
`safeName` => `name`
`userName` => `username`
`roleName` => `name`

Reply:
Renamed safeName as name in the request/response payloads and related fields. All other places renamed to safename. roleName renamed to rolename, as it is referenced in the same place where the safe name also used.